app/build.gradle

```diff
@@ -221,11 +221,14 @@ android {
 }
 
 dependencies {
+    implementation project(':content-descriptions')
 
-    implementation("com.google.dagger:hilt-android:$daggerHiltVersion")
-    ksp("com.google.dagger:hilt-compiler:$daggerHiltVersion")
     ksp("androidx.hilt:hilt-compiler:$jetpackHiltVersion")
+    ksp("com.google.dagger:hilt-compiler:$daggerHiltVersion")
+    ksp("com.github.bumptech.glide:ksp:$glideVersion")
 
+    implementation 'androidx.compose.material3:material3-android:1.2.1'
```

Comment:
This is already declared further down line 364

Reply:
Removed the on on line ~231 as the line 364 has other compose-related stuff around it so is a more natural place for it.